src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetManager.kt

```diff
@@ -0,0 +1,20 @@
+package zd.zero.waifu.motivator.plugin.assets
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+object AudibleAssetManager : RemoteAssetManager<AudibleMotivationAssetDefinition>(
+    AssetCategory.AUDIBLE,
+    emptyList()
+) {
+    override fun applyAssetUrl(
+        visualAsset: AudibleMotivationAssetDefinition,
+        assetUrl: String
+    ): AudibleMotivationAssetDefinition =
+        visualAsset.copy(path = assetUrl)
+
+    override fun convertToDefinitions(defJson: String): List<AudibleMotivationAssetDefinition> =
+        Gson().fromJson<List<AudibleMotivationAssetDefinition>>(
```

Comment:
I had to do this because of type erasure (I think).
Gson wasn't able to get the type from the generic when it was in the abstract class, so It had to be moved up to the implementing class.